schemas/common-1.json

```diff
@@ -5,7 +5,7 @@
   "definitions": {
     "query": {
       "type": "string",
-      "pattern": "(?i)^(?:select|explain(?!\\s+analyze(?!\\s+select)))(?:(?:\\/\\*.*\\*\\/)|(?:[\\\"\\'][^\\\"\\']*[\\\"\\'])|(?:--.*\\n)|[^;\\*])+;$"
+      "pattern": "(?i)^(?:analyze|reindex|vacuum|select|explain(?!\\s+analyze(?!\\s+select)))(?:(?:\\/\\*.*\\*\\/)|(?:[\\\"\\'][^\\\"\\']*[\\\"\\'])|(?:--.*\\n)|[^;\\*])*;$"
```

Comment:
Not a fan of that final `*` as that may lead to non-complete sql statements.

If you want to allow bare `ANALYZE;` you may want to contemplate that case separately...